### ISSUE TITLE
import Multi-line curl error 

### ISSUE THREAD

**Issue by kackerx** [getinsomnia/insomnia#6065]
### Expected Behavior

import edge or postman's curl

### Actual Behavior

Error while scanning for resources to improt: cannot read properties of undefined (reading 'trim') ??

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

latest

### What operating system are you using?

macOS

### Operating System Version

macos 13.1

### Installation method

insomnia

**Comment by jackkav** [comment]
Thanks for the report, could you provide the import file? or more information about how you created it.

**Comment by kackerx** [comment]
> Thanks for the report, could you provide the import file? or more information about how you created it.

sure, such as
```bash
curl --location --request POST 'http://localhost:8900/fbi/contract/list' \
--header 'x-rpc-fake_user: miyin' \
--header 'x-rpc-god;' \
--header 'User-Agent: Apifox/1.0.0 (https://www.apifox.cn)' \
--header 'Content-Type: application/json' \
--header 'Accept: */*' \
--header 'Host: localhost:8900' \
--header 'Connection: keep-alive' \
--data-raw '{
    "page": 1,
    "page_size": 100,
    "name": "",
    "company_code_list": [],
    "supplier_code_list": [],
    "currency_list": [],
    "usable": false
}'
```
<img width="817" alt="image" src="https://github.com/Kong/insomnia/assets/55371516/2e71a9c1-124c-46ec-8ad2-4ab06c94c2fc">
I'm not sure if it was my mishandling

**Comment by jackkav** [comment]
ah you can paste that directly into the url bar of an empty request and it will be parsed. You are correct though, it does list cURL as one of the import types. Thats our bad. Thanks again.

**Comment by kackerx** [comment]
> ah you can paste that directly into the url bar of an empty request and it will be parsed. You are correct though, it does list cURL as one of the import types. Thats our bad. Thanks again.

![image](https://github.com/Kong/insomnia/assets/55371516/dfc9480e-0c81-4fda-81b5-8196ebc5756b)
You mean this...? Seems to be unsupportive

**Comment by jackkav** [comment]
ah its because of this line, 
`--header 'x-rpc-god;' \`
it works if you remove it.
although the code should support valueless headers, so this is a bug

**Comment by kackerx** [comment]
> ah its because of this line, `--header 'x-rpc-god;' \` it works if you remove it. although the code should support valueless headers, so this is a bug

ah. Thank you very much. I got it!

**Comment by jackkav** [comment]
Fixed in #6069
